src/arguments.cpp

```diff
@@ -86,6 +87,9 @@ Error Arguments::parse(const char* args) {
         } else if (strcmp(arg, "flamegraph") == 0 || strcmp(arg, "svg") == 0) {
             _dump_flamegraph = true;
             _counter = value == NULL || strcmp(value, "samples") == 0 ? COUNTER_SAMPLES : COUNTER_TOTAL;
+        } else if (strcmp(arg, "calltree") == 0) {
```

Comment:
Do you mind if we rename the option to "tree"? Just to type fewer letters in the command line :)
Also profiler.sh can automatically select this option if the filename ends with .html, similarly to how we select flamegraph output if the filename ends with .svg.

Reply:
I have changed calltree to tree and submitted changes.